add initial mssm import (#163)
greenelab/covid19-review: content/95-ismms-himc-appendix.md

more specific metrics of disease severity or time course. Noting
mortality, time from disease onset, pre-existing conditions, or severity
of lung pathology in post-mortem tissue samples would paint a fuller
picture of how to assess risk level and the relationship between
severity of disease and immunopathology. Another limitation is the
selection of cytokines and immune markers for analysis, as the selection
criteria were based on the cell subsets and cytokine storm typically
seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and
immune profiling may reveal novel therapeutic targets that were not
included in this study.

#### Significance

This study identifies potential therapeutic targets
that could prevent acute respiratory disease syndrome (ARDS) and
mortality in patients most severely affected by COVID-19. The authors
propose testing monoclonal antibodies against IL6-R or GM-CSF to block
recruitment of inflammatory monocytes and the subsequent cytokine storm
in these patients.

#### Credit

*Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*

> a\. Title of the article : **Clinical Characteristics of 2019 Novel

Reviewer: ```suggestion
### Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis
```

> Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis**

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion

[@doi:10.1101/2020.02.14.20021535]

```

> b\. Small number of Immunology Keywords (will be used in STEP 3)

Reviewer: ```suggestion
#### Keywords
```

> 

Reviewer: ```suggestion
```

> White Blood Cells, Lymphocytes, Neutrophils

Reviewer: ```suggestion
- White Blood Cells
- Lymphocytes
- Neutrophils
```

> c\. A few lines that summarize the main findings of the article (the

Reviewer: ```suggestion
#### Main Findings
```

> shorter the better!)

Reviewer: ```suggestion
```

The authors performed a meta analysis of literature on clinical,
laboratory and radiologic characteristics of patients presenting with
pneumonia related to SARSCoV2 infection, published up to Feb 6 2020.
They found that symptoms that were mostly consistent among studies were
sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise
and muscle pain were highly variable across studies. Leukopenia (mostly
lymphocytopenia) and increased white blood cells were highly variable
across studies. They identified three most common patterns seen on CT
scan, but there was high variability across studies. Consistently across
the studies examined, the authors found that about 75% of patients need
supplemental oxygen therapy, about 23% mechanical ventilation and about
5% extracorporeal membrane oxygenation (ECMO). The authors calculated a
staggering pooled mortality incidence of 78% for these patients.

> d\. A critical analysis that explicit the limitations of the study, if

Reviewer: ```suggestion
#### Limitations
```

> any.

Reviewer: ```suggestion
```

Reviewer: noticed this one seems to have fixed-width lines

The authors mention that the total number of studies included in this
meta analysis is nine, however they also mentioned that only three
studies reported individual patient data. It is overall unclear how many
patients in total were included in their analysis. This is mostly
relevant as they reported an incredibly high mortality (78%) and mention
an absolute number of deaths of 26 cases overall. It is not clear from
their report how the mortality rate was calculated.

The data is based on reports from China and mostly from the Wuhan area,
which somewhat limits the overall generalizability and applicability of
these results.

> e\. Importance and implications of these findings in the context of the

Reviewer: ```suggestion
#### Significance
```

> current epidemics

Reviewer: ```suggestion
```

This meta analysis offers some important data for clinicians to refer to
when dealing with patients with COVID-19 and specifically with
pneumonia. It is very helpful to set expectations about the course of
the disease.

> 

Reviewer: ```suggestion
#### Credit

*Review as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*

```

#### Significance

The authors identified potential T-cell and B-cell epitopes
that may be good candidates for peptide based vaccines against CoV2.
They also made interesting observations in comparing SARS-CoV and CoV2
potential antibody binding sites, noting that SARS-CoV had larger attack
surfaces for potential neutralizing antibody binding. One of the
highlights of this paper was the authors’ mutation analysis of 68 viral
genomes from 4 continents. This analysis not only validated their
computational method for identifying T-cell epitopes, but showed that
immune surveillance likely drives viral mutation in MHC-I binding
peptides. The smaller attack surface may point to potential mechanisms
of immune evasion by CoV2. However, absence of mutations in the RBD of
CoV2 and the small number of mutations in peptides presentable to T
cells suggests that vaccines against multiple epitopes could still
elicit robust immunity against CoV2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*

### Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein

> 

Reviewer: ```suggestion

[@doi:10.1016/j.cell.2020.02.058]

```

prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.

#### Significance

This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*

### The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis

> 

Reviewer: ```suggestion

[@doi:10.1101/2020.02.26.20026989]

```

 
#### Significance

Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2.  Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option.  An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients.  Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle.  The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.  
 
 

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

viral proteins in HEK293T cells. This precludes any interactions between
the viral proteins, or the combined effects of multiple proteins on the
host, as they are expressed individually. Moreover, HEK293T cells come
from primary embryonic kidney and therefore might not reflect how
SARS-CoV-2 interacts with its primary target, the lung. However, the
authors found that the proteins found to interact with viral proteins in
their experiments are enriched in lung tissue compared to HEK293Ts.

#### Significance

The authors provide a “SARS-CoV-2 interaction map,”
which may provide potential hypotheses as to how the virus interacts
with the host. Further, they identified existing drugs that could
disrupt these host-viral interactions and curb SARS-CoV-2 infection.
Although these interactions have not been validated, this paper acts as
a valuable resource.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*

### First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients

> 

Reviewer: ```suggestion

[@doi:0.1101/2020.03.22.20034041]

```

-   Study analyzed bulk and scRNAseq data of olfactory cell types from
    publicly-available mouse, nonhuman primate and human datasets.

-   show that *ACE2* and *TMPRSS2* (genes involved in SARS-CoV-2 entry)
    are expressed in olfactory epithelial (OE) cells, basal stem cells
    and respiratory epithelium (RE), but not sensory neurons.

-   Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
    et al. 2020) revealed that *ACE2* and *TMPRSS2* expression in OE
    sustentacular cells was similar to expression in the remainder of
    the non-nasal respiratory tract.

#### Limitations

-   Transcript data alone from healthy respiratory/olfactory cells is
    not sufficient to confirm infectivity of nasal passage, or to
    indicate damage to epithelia.

-   No mechanism defined for anosmia; it is not clear if epithelial
    injury leads to reduced sensitivity or increased inflammation and
    altered immune contexture drives neural/epithelial dysfunction. Will
    be critical to test this in CoVID-19 patient samples or mouse
    models.

#### Significance

-   Study provides possible rationale for anosmia observed in several
    CoVID-19 patients.

> -  

Reviewer: ```suggestion
### Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis
```

> -  

Reviewer: ```suggestion
```

> -  

Reviewer: ```suggestion
```

> -  

Reviewer: ```suggestion
[@doi:10.1101/2020.03.31.019216]

```

> -  

Reviewer: ```suggestion

[@doi:10.1101/2020.03.28.20045765]

```

[@doi:10.1101/2020.03.25.009084]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- anosmia
- olfaction
- scRNAseq

#### Main Findings

-   Study analyzed bulk and scRNAseq data of olfactory cell types from
    publicly-available mouse, nonhuman primate and human datasets.

-   show that *ACE2* and *TMPRSS2* (genes involved in SARS-CoV-2 entry)
    are expressed in olfactory epithelial (OE) cells, basal stem cells
    and respiratory epithelium (RE), but not sensory neurons.

-   Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
    et al. 2020) revealed that *ACE2* and *TMPRSS2* expression in OE
    sustentacular cells was similar to expression in the remainder of
    the non-nasal respiratory tract.

#### Limitations

-   Transcript data alone from healthy respiratory/olfactory cells is
    not sufficient to confirm infectivity of nasal passage, or to
    indicate damage to epithelia.

-   No mechanism defined for anosmia; it is not clear if epithelial
    injury leads to reduced sensitivity or increased inflammation and
    altered immune contexture drives neural/epithelial dysfunction. Will

>     be critical t

Reviewer: ```suggestion
#### Limitations
```

Reviewer: This is not showing up correctly, very weird. Basically there is a "Limitations" with a `###` header (rather than `####`)

contributes to greater disease severity. Does the increased ACE-2
expression lead to greater infectivity with SARS-CoV-2? There is no
mechanistic explanation for why ACE-2 levels are increased in COPD
patients. The authors could also have considered the impact of
co-morbidities and interventions such as corticosteroids or
bronchodilators on ACE-2 expression. Finally, given the extensive
sequencing performed, the authors could have conducted significantly
more in-depth analyses into gene signature differences.

#### Significance

This study attempts to address an
important clinical finding that both smokers and COPD patients show
increased mortality from COVID-19. The novel finding that ACE-2
expression is induced in smokers and COPD patients suggests not only a
mechanism for the clinical observation, but also highlights the
potential benefit of smoking cessation in reducing the risk of severe
COVID-19 disease.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*

**Review by Katherine Lindblad and Tamar Plitt**
10.1101/2020.03.18.20038513

> Title: Dynamic profile of severe or critical COVID-19 cases

Author: Done!
